radix-engine-interface/src/blueprints/utils.rs

```diff
@@ -0,0 +1,99 @@
+use crate::blueprints::access_controller::*;
+use crate::blueprints::account::*;
+use crate::blueprints::identity::*;
+use crate::blueprints::package::*;
+use crate::blueprints::pool::*;
+use crate::internal_prelude::*;
+
+/// This function resolves the [`BlueprintId`] based on the entity type for native blueprints.
+///
+/// If the entity type is [`EntityType::GlobalGenericComponent`] then this function returns [`None`]
+/// since it can't tell what the [`BlueprintId`] of a generic component is. Otherwise, in the case
+/// where the [`EntityType`] belongs to a native blueprint this function will translate it into the
+/// appropriate [`BlueprintId`].
+pub fn resolve_main_module_blueprint_id(entity_type: EntityType) -> Option<&'static BlueprintId> {
```

Comment:
SUPER MINOR / OPTIONAL: For discoverability, I'd even support adding this as a method on `EntityType`

Reply:
That was my initial thinking but we unfortunately can't do that since the blueprint name constants that we need are defined in the `radix-engine-interface` crate.